fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -5254,6 +5254,8 @@ keyword ::=
     {: RESULT = id; :}
     | KW_RESOURCES:id
     {: RESULT = id; :}
+    | KW_RESOURCE_GROUP:id
```

Comment:
why not use `RESOURCE GROUP` rather than `resource_group`

Reply:
And when will `resource_group` be used?

Reply:
Current SQL command is `CREATE RESOURCE_GROUP, ALTER RESOURCE_GROUP`, which is pretty strange...

The `resource_group` is also used as a variable, like `set resource_group = 'rg1'`.

Reply:
Actually we should use `CREATE RESOURCE GROUP` instead of `CREATE RESOURCE_GROUP`. Cause no other SQL keyword contains a underline.

But since the 2.2 version introduce this keyword and our user will use it, maybe we don't have chance to fix this mistake.

Reply:
done